posthog/api/query.py

```diff
@@ -61,7 +61,7 @@ def get_throttles(self):
         if self.action in ("draft_sql", "chat"):
             return [AIBurstRateThrottle(), AISustainedRateThrottle()]
         if query := self.request.data.get("query"):
-            if query.get("kind") == "HogQLQuery":
+            if isinstance(query, dict) and query.get("kind") == "HogQLQuery":
```

Comment:
we also have a util `get_from_dict_or_attr` but guess it's guaranteed to be dict here? :)